service/tls.go

```diff
@@ -242,3 +249,53 @@ func isValidDuration(durationString string) (bool, error) {
 	}
 	return false, nil
 }
+
+// cipherSuites returns list of available cipher suites
+func cipherSuites(dcs []string) ([]uint16, error) {
+	var (
+		availableCipherSuites     []uint16
+		availableCipherSuitesName []string
+	)
+
+	ciphers := defaultCipherSuitesMap()
+	if len(dcs) != 0 {
+		for _, cipher := range dcs {
+			if _, ok := ciphers[cipher]; ok {
+				delete(ciphers, cipher)
+			} else {
+				err := fmt.Errorf("Invalid cipher suite: %s", cipher)
```

Comment:
```suggestion
				err := glg.Errorf("Invalid cipher suite: %s", cipher)
```
use log library

Reply:
fixed in [9365b69](https://github.com/AthenZ/authorization-proxy/pull/18/commits/9365b69727228bd3b1cedcf9ccd09dd17ecf7611)